composer.json

```diff
@@ -11,11 +11,11 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
+        "php": "^7.2",
```

Comment:
^7.2|^8.0 Maybe add php8 support? The 8.0.0 final release is still scheduled for next Thursday, November 26th.

Reply:
Will add 8.0 support when a stable version is release. I'm also planning on dropping 7.2 since it's EOL on Nov 30